libqtile/backend/wayland/core.py

```diff
@@ -393,6 +394,15 @@ def get_enabled_outputs(self) -> list[Output]:
     def name(self) -> str:
         return "wayland"
 
+    @property
+    def active_keyboard(self) -> inputs.Keyboard | None:
+        keyboard = self.seat.get_keyboard()
```

Comment:
Oh wait can you check if this is None before the for loop?

Reply:
Otherwise we would get NoneType has no attribute ._ptr inside the for loop

Reply:
Yeah. Just thought of that after I pushed this.

Reply:
Fine now thanks